runtime/src/accounts.rs

```diff
@@ -785,21 +786,51 @@ impl Accounts {
         bank_id: BankId,
         index_key: &IndexKey,
         filter: F,
-        config: ScanConfig,
+        config: &ScanConfig,
```

Comment:
What was the motivation for switching from move to reference here?

Reply:
So I can check config.is_aborted() after calling a scan without cloning the config.